src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BaseFunctionalTest.java

```diff
@@ -50,44 +51,41 @@ public void setUp() throws Exception {
         this.isProxyEnabled = Boolean.valueOf(config.getString("proxyout.enabled"));
         this.fluxFuncTest = config.getBoolean("flux-func-test");
 
-        StorageCredentialsAccountAndKey storageCredentials =
-            new StorageCredentialsAccountAndKey(
-                config.getString("test-storage-account-name"),
-                config.getString("test-storage-account-key")
-            );
+        String connectionString = String.format(
+            "DefaultEndpointsProtocol=https;BlobEndpoint=%s;AccountName=%s;AccountKey=%s",
+            config.getString("test-storage-account-url"),
+            config.getString("test-storage-account-name"),
+            config.getString("test-storage-account-key")
+        );
+        leaseClientProvider =  blobClient -> new BlobLeaseClientBuilder().blobClient(blobClient).buildClient();
+        BlobServiceClientBuilder blobServiceClientBuilder = new BlobServiceClientBuilder()
+            .connectionString(connectionString);
```

Comment:
we can still use `StorageSharedKeyCredential` here and set `url` with `builder.endpoint(url)`